### DISCUSSION TITLE
Website constantly going down?

### DISCUSSION THREAD

**Discussion by AdamT20054** [simple-login/app#1507] [Q&A]
The site is CONSTANTLY going down, has been for about 2 hours now. It'll work for for about 5 minutes, and then it'll load forever and spit out a "The connection has timed out" error.

The status page indicates nothing has wrong, tried multiple different IP's, and even tried it on my phone. All with the same result.

Can't find anywhere else to post it, so I'm posting it here,

**Comment by AdamT20054** [comment]
still constantly going down..
Even the extension can't get a connection

**Reply by nguyenkims** [reply]
From our status page https://status.simplelogin.io/, the service is quite stable recently. I wonder if you have a firewall, a DNS, etc. that might block SimpleLogin?

**Reply by AdamT20054** [reply]
> From our status page https://status.simplelogin.io/, the service is quite stable recently. I wonder if you have a firewall, a DNS, etc. that might block SimpleLogin?

That's exactly why i tried different devices, IPs, browsers and I even tried it on my Gyms network when I went there.

Looks to be stable now buy yesterday it was a farce to even get my domain linked.

**Comment by tuxiqae** [comment]
I have a similar issue and I think that for some reason  I'm being blocked by the platform.
Once I regenerate a new public IP everything works for 2 mins until it happens again.

**Reply by nguyenkims** [reply]
When that happens again, can you please take a screenshot of your browser network tab (cf  https://share.cleanshot.com/vnJcsNYS ) and paste it here?
